lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -112,6 +112,11 @@ def trigger
     @triggers
   end
 
+  def command_insync?(current, should)
+    should_value = File.expand_path(should[0]).gsub(/\//, '\\')
```

Comment:
Puppet::Property is inherently multivalued but often it's used to manage a single value. So you'll see lots of things like:

```ruby
should.all? { |value| value == .. }
```